HW2_Slepov/calculator.py

```diff
@@ -0,0 +1,35 @@
+def multiplication(b):
+    a1 = float(b[0])
+    a2 = float(b[2])
+    return a1*a2
+def addition(b):
+    a1 = float(b[0])
+    a2 = float(b[2])
+    return a1+a2
+def substraction(b):
+    a1 = float(b[0])
+    a2 = float(b[2])
+    return a1-a2
+def division(b):
+    a1 = float(b[0])
+    a2 = float(b[2])
+    return a1/a2
+def main(b):
+    if b[1]=="*":
```

Comment:
```suggestion
    if b[1] == "*":
```
Оператор сравнения тоже отделяется пробелами 😌